src/plugins/intel_gpu/src/kernel_selector/core/common/kernel_base_opencl.cpp

```diff
@@ -92,14 +92,24 @@ std::pair<std::string, std::string> KernelBaseOpenCL::CreateJit(const std::strin
         .add_line("// Kernel name: " + kernel_id)
         .value_macro("KERNEL(name)", "__kernel void " + kernel_id)
         .decoration_macro("FUNC", "", kernel_id)
-        .decoration_macro("FUNC_CALL", "", kernel_id);
+        .decoration_macro("FUNC_CALL", "", kernel_id)
+        .decoration_macro("CONST_ARRAY_DECL", "__constant size_t ", kernel_id + " []")
+        .decoration_macro("CONST_ARRAY_REF", "", kernel_id);
 
     undefs += "#undef KERNEL\n";
     undefs += "#undef FUNC\n";
     undefs += "#undef FUNC_CALL\n";
+    undefs += "#undef CONST_ARRAY\n";
```

Comment:
Please check if my suggestion is right.

```suggestion
    undefs += "#undef CONST_ARRAY_DECL\n";
    undefs += "#undef CONST_ARRAY_REF\n";
```

Reply:
Thanks ! I missed my change :D